public/components/event_analytics/explorer/sidebar/field_insights.tsx

```diff
@@ -103,14 +103,14 @@ export const FieldInsights = ({ field, query }: any) => {
           });
         }
       })
-      .catch((error) => console.error(error));
+      .catch((error) => {});
   }, []);
 
-  const getInsights = async (query: string) => {
+  const getInsights = async (queryStr: string) => {
```

Comment:
q: type is already `string`, why the renaming here?

Reply:
This is because there is 'query' in the scope which is the query state, the query here is shadowing the global one and eslint throws error.

Reply:
addressed in [79862f7](https://github.com/opensearch-project/dashboards-observability/pull/324/commits/79862f72c1d35a131525501fe2d32e8974312924)